plugins/pipeline-template-validator.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const schema = require('screwdriver-data-schema');
+const templateSchema = schema.api.templateValidator;
+const pipelineValidator = require('screwdriver-template-validator').parsePipelineTemplate;
+
+/**
+ * Hapi Pipeline Template Validator Plugin
+ *  - Validates sd-template.yaml and returns the parsed pipeline template with any
+ *    errors associated with it
+ * @method register
+ * @param  {Hapi.Server}    server
+ * @param  {Function} next
+ */
+const pipelineTemplateValidatorPlugin = {
+    name: 'pipeline-template-validator',
+    async register(server) {
+        server.route({
+            method: 'POST',
+            path: '/validator/pipelineTemplate',
```

Comment:
this should be changed to
```suggestion
            path: '/pipeline/template/validate',
```

Reply:
Move the API under  pipelines/templates/